__tests__/components/pages/__snapshots__/PlantInfo_test.js.snap

```diff
@@ -1,9 +1,190 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`PlantInfo Page PlantInfo Page does render 1`] = `
-<View>
-  <Text>
-    PlantInfo Component
-  </Text>
-</View>
+<RCTSafeAreaView
+  emulateUnlessSupported={true}
+  style={
+    Object {
+      "alignItems": "center",
+      "backgroundColor": "#fff",
+      "flex": 1,
+      "justifyContent": "center",
+    }
+  }
```

Comment:
Are we planning on taking the styling from the PlantInfo component and putting it in the stylesheet later?